data/human/boarding missions.txt

```diff
@@ -703,3 +703,260 @@ mission "Turn in Tarquin"
+			choice
+				`	(Repair the hull next.)`
+					goto hull(e)
+				`	(Repair the power next.)`
+					goto power(e)
+			label engines(h)
+			branch misfire
+				random < 60
+			`	As you are fixing the engine systems a valve bursts open. Luckily the secondary valve is mechanical and doesn't need power, allowing it to close itself automatically and prevent the coolant from forcefully spraying out. With more caution, you finish up your repairs of the engines.`
+			choice
+				`	(Repair the power.)`
+					goto power(eh)
+				`	(Flee the ship for fear my repairs have made the ship more dangerous.)`
+					goto flee
+			label engines(p)
+			`	As you are fixing the engine systems a valve bursts open. Luckily you restored power, allowing the secondary valve to close itself automatically and prevent the coolant from forcefully spraying out. With more caution, you finish up your repairs of the engines.`
+			choice
+				`	(Repair the hull.)`
+					goto hull(ep)
+				`	(Flee the ship for fear my repairs have made the ship more dangerous.)`
+					goto flee
+			label engines(hp)
+			`	As you are fixing the engine systems a valve bursts open. Luckily you restored power, allowing the secondary valve to close itself automatically and prevent the coolant from forcefully spraying out. With more caution, you finish up your repairs of the engines.`
+			`	With the repairs finished you head to the cockpit to see if the <origin> starts without exploding. Prepared to bolt for the airlock should things go wrong, you start the ship.`
+			branch fail
+				random < 5
+			action
+			`	The ship thrums to life. With bated breath you wait for the sound of some system exploding, but nothing happens. Success! The ship is yours.`
+				goto success
+			label power(h)
+			branch powersurge
+				random < 5
+			`	With a few simple repairs you manage to get the ship's power systems operational and running smoothly.`
+			choice
+				`	(Repair the engines.)`
+					goto engines(hp)
+				`	(Flee the ship for fear my repairs have made the ship more dangerous.)`
+					goto flee
+			label power(e)
+			branch powersurge
+				random < 55
+			`	With a few simple repairs you manage to get the ship's power systems operational and running smoothly.`
+			choice
+				`	(Repair the hull.)`
+					goto hull(ep)
+				`	(Flee the ship for fear my repairs have made the ship more dangerous.)`
+					goto flee
+			label power(eh)
+			branch powersurge
+				random < 15
+			`	With a few simple repairs you manage to get the ship's power systems operational and running smoothly.`
+			`	With the repairs finished you head to the cockpit to see if the <origin> starts without exploding. Prepared to bolt for the airlock should things go wrong, you start the ship.`
+			branch fail
+				random < 25
+			action
```

Comment:
This action is empty. Was there intended to be something here, or should this be removed?

Reply:
Looks like there were a couple of these left in the branching from a different method I tried, they've been removed.